### PR TITLE
Audit To-Hit Values of Kitchen Knives

### PR DESCRIPTION
**Pull request by ANickelN**
#### Summary
Balance "Audit To-Hit Values of Kitchen Knives"

#### Purpose of change

To adjust the to-hit values of kitchen knives to match game_balance.md

#### Describe the solution

Go through knives_kitchen.json and correct values according to game_balance.md

#### Describe alternatives you've considered

Not correcting anything

#### Testing

Spawn in changed items in-game and note corrected values.

#### Additional context

This is a very minor change and only effected the meat cleaver and bread knife.

### CODE REVIEW

File: tests/effective_dps_test.cpp [hunk 1]
```diff
@@ -529,7 +529,7 @@ static void check_knives( const std::function<Approx( const std::string & )> &ca
         CHECK( calc_expected_dps( "knife_hunting" ) == 11.0 );
         CHECK( calc_expected_dps( "kirpan_cheap" ) == 10.5 );
         CHECK( calc_expected_dps( "switchblade" ) == 10.5 );
-        CHECK( calc_expected_dps( "knife_meat_cleaver" ) == 14.5 );
+        CHECK( calc_expected_dps( "knife_meat_cleaver" ) == 17.22 );
```
Comment (anothersimulacrum) [inline tests/effective_dps_test.cpp:532]: You should drop the damage or otherwise make adjustments, this is knife has no reason to have a DPS this high.

Comment (ANickelN) [reply tests/effective_dps_test.cpp:532]: Dropped the damage of the meat cleaver to match the vegetable cleaver at 4 bash and 15 cut

**Comment by ANickelN** [comment]
Failure is unrelated.

File: data/json/items/melee/knives_kitchen.json [hunk 2]
```diff
@@ -164,9 +164,9 @@
     "longest_side": "30 cm",
     "price": 3500,
     "price_postapoc": 50,
-    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "uneven" },
-    "bashing": 6,
-    "cutting": 20,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "neutral" },
+    "bashing": 4,
+    "cutting": 15,
```
Comment (I-am-Erk) [inline data/json/items/melee/knives_kitchen.json:169]: Any reason to make this less effective than a chef knife? A cleaver is pretty hefty.

Comment (ANickelN) [reply data/json/items/melee/knives_kitchen.json:169]: It was 20 before, @anothersimulacrum requested to lower the damage so I brought it in-line with the vegetable cleaver.

Comment (I-am-Erk) [reply data/json/items/melee/knives_kitchen.json:169]: If that's consistent with the veg it is probalby ok.

**Pull request merged**